pkgs/development/cuda-modules/setup-hooks/extension.nix

```diff
@@ -32,31 +47,33 @@ final: _: {
       {}
     );
 
-  autoAddOpenGLRunpathHook =
+  autoAddDriverRunpath =
```

Comment:
We should also keep an alias around at least for one year. Unfortunately, I don't think `aliases.nix` works with nested package sets and we can't show warnings when evaluating pubilc attributes, so we'll have to just set `autoAddOpenGLRunapthHook = autoAddDriverRunpath;` and rely on release notes/documentation to communicate the change